README.md

```diff
@@ -55,7 +56,7 @@ gsub.publish('fruit', new TextEncoder().encode('banana'))
 
 ```js
 const options = {…}
-const gossipsub = new Gossipsub(libp2p, options)
+const gossipsub = gossipsub(options)(libp2p)
```

Comment:
maybe makes more sense to put example where to pass `gossipsub(options)` in libp2p init